<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-09-17T21:10:30 IST" name="Default suite" finished-at="2022-09-17T21:10:59 IST" duration-ms="29407">
    <groups>
    </groups>
    <test started-at="2022-09-17T21:10:30 IST" name="Default test" finished-at="2022-09-17T21:10:59 IST" duration-ms="29407">
      <class name="testpakage.TestClass_Testng">
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:testpakage.TestClass_Testng@74751b3]" started-at="2022-09-17T21:10:30 IST" name="openBrowser" finished-at="2022-09-17T21:10:47 IST" duration-ms="17003" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method signature="verifyLogInfunctionality()[pri:0, instance:testpakage.TestClass_Testng@74751b3]" started-at="2022-09-17T21:10:47 IST" name="verifyLogInfunctionality" finished-at="2022-09-17T21:10:54 IST" duration-ms="6856" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at testpakage.TestClass_Testng.verifyLogInfunctionality(TestClass_Testng.java:71)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogInfunctionality -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:testpakage.TestClass_Testng@74751b3]" started-at="2022-09-17T21:10:54 IST" name="afterMethod" finished-at="2022-09-17T21:10:55 IST" duration-ms="850" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLogInfunctionality status=FAILURE method=TestClass_Testng.verifyLogInfunctionality()[pri:0, instance:testpakage.TestClass_Testng@74751b3] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method signature="verifyHomepage()[pri:1, instance:testpakage.TestClass_Testng@74751b3]" started-at="2022-09-17T21:10:55 IST" name="verifyHomepage" finished-at="2022-09-17T21:10:58 IST" duration-ms="3476" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyHomepage -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:testpakage.TestClass_Testng@74751b3]" started-at="2022-09-17T21:10:58 IST" name="afterMethod" finished-at="2022-09-17T21:10:58 IST" duration-ms="114" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyHomepage status=SUCCESS method=TestClass_Testng.verifyHomepage()[pri:1, instance:testpakage.TestClass_Testng@74751b3] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterclass()[pri:0, instance:testpakage.TestClass_Testng@74751b3]" started-at="2022-09-17T21:10:58 IST" name="afterclass" finished-at="2022-09-17T21:10:59 IST" duration-ms="1043" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterclass -->
      </class> <!-- testpakage.TestClass_Testng -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
